recipes-wpe/wpeframework/include/streamer.inc

```diff
@@ -1,19 +1,41 @@
-# Backporting streamer pacakge config, this needs to go upstream meta-wpe once we sync up with the latest
-STREAMER_IMPLEMENTATION ?= "QAM"
-STREAMER_FRONTENDS      ?= "1"
-STREAMER_DECODERS       ?= "2"
-STREAMER_STANDARD       ?= "DVB"
-STREAMER_SI_PARSING     ?= "NO"
-STREAMER_TS_SCANNING    ?= "NO"
-STREAMER_HOME_TS        ?= ""
-
-PACKAGECONFIG[streamer] = " \
-    -DPLUGIN_STREAMER=ON \
-    -DSTREAMER_IMPLEMENTATION=${STREAMER_IMPLEMENTATION} \
-    -DPLUGIN_STREAMER_FRONTENDS=${STREAMER_FRONTENDS} \
-    -DPLUGIN_STREAMER_DECODERS=${STREAMER_DECODERS} \
-    -DPLUGIN_STREAMER_STANDARD=${STREAMER_STANDARD} \
-    -DPLUGIN_STREAMER_SI_PARSING=${STREAMER_SI_PARSING} \
-    -DPLUGIN_STREAMER_TS_SCANNING=${STREAMER_TS_SCANNING} \
-    -DPLUGIN_STREAMER_HOME_TS=${STREAMER_HOME_TS} \
-    ,,"
+# Streamer supports QAM or AAMP implementation
+# Add 'QAM' and/or 'Aamp' to STREAMER_IMPLEMENTATIONS
+STREAMER_IMPLEMENTATIONS ?= "Aamp"
```

Comment:
dont use mixed casing `aamp` is less confusing for bitbake

Reply:
@kraj  Folder names used in Streamer Plugin are Aamp and QAM Please check below link (https://github.com/WebPlatformForEmbedded/ThunderNanoServices/tree/master/Streamer/Implementation)     so set the same folder names in recipe

Reply:
Really ? This seems odd naming convention probably needs to be addressed but this patch is fine then